rtsp.go

```diff
@@ -215,11 +259,85 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 			}
 		}
 	})
-	_, err = rc.client.Play(nil)
+
+	return nil
+}
+
+// InitH265
+func (rc *rtspCamera) InitH265(tracks media.Medias, baseURL *url.URL) (err error) {
+	var format *formats.H265
+
+	track := tracks.FindFormat(&format)
+	if track == nil {
+		rc.logger.Warnf("tracks available")
+		for _, x := range tracks {
+			rc.logger.Warnf("\t %v", x)
+		}
+		return errors.New("h265 track not found")
+	}
+
+	_, err = rc.client.Setup(track, baseURL, 0, 0)
 	if err != nil {
 		return err
 	}
-	clientSuccessful = true
+
+	rtpDec, err := format.CreateDecoder2()
+	if err != nil {
+		rc.logger.Errorf("error creating H265 decoder %v", err)
+		return err
+	}
+
+	rc.rawDecoder, err = newDecoder(C.AV_CODEC_ID_HEVC)
+	if err != nil {
+		return err
+	}
+
+	// For H.265, handle VPS, SPS, and PPS
```

Comment:
Is there anything we're not handling? Call out in the comment.

Reply:
- Added verbose warning logs for parameter sets.
- Still need to test out wether decoders can recover if not available.